build.gradle

```
@@ -108,7 +108,7 @@ testing {
 
 tasks.withType(Test).configureEach {
     finalizedBy jacocoTestReport
-    systemProperty "mongo.image", "mongo:4.4.28-focal"
+    systemProperty "mongo.image", "7.0.15-jammy"
```

In fact, what was expected here was
```suggestion
    systemProperty "mongo.image", "mongo:7.0.15-jammy"
```